doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -283,8 +283,33 @@ In Chapter 3, Figure 3.2 shows a high level Virtualised OpenStack services topol
 
 <a name="4.4.1"></a>
 ### 4.4.1. Support for Profiles and T-shirt instance types
-**Content to be developed**
-Develop on Flavors and metadata
+Reference Model Chapter 4 and  5 provide information about the instance types and sizing information. To implement these profiles and sizes requires the setting up of information as specified in the Tables below. As OpenStack no longer supports default flavors, the CNTT pre-defined flavors will have to be created with their various configuration properies.
```

Comment:
```...provide information about the instance types and sizing information``` -> ```...provide information about the instance types and size```

Reply:
@msidana Done (although RM Chapters 4 and 5 refer to sizing)